README.md

```diff
@@ -92,42 +110,43 @@ let authenticationParameters = AuthenticationParameters(
     scheme: scheme)
 
-checkout3DS.authenticate(authenticationParameters: authenticationParameters) { error in
-    if let error = error {
-        // Handle error.
-    } else {
-        // Continue with payment.
-    }
+checkout3DS.authenticate(authenticationParameters: authenticationParameters) { result in
+    switch authenticationResult {
+    case .success(let authenticationResult):
+        // handle authentication result. Checkout Payment Authorisation section.
+    case .failure(let error):
+        // handle failure scenarios
+     }
 }
 ```
 👉&nbsp;&nbsp;[See the Integration Guide with Code Examples](https://www.checkout.com/docs/risk-management/3d-secure/sessions/non-hosted-sessions/3d-secure-mobile-sdks) for full details.
 
 📚&nbsp;&nbsp;[Read the reference documentation](https://checkout.github.io/checkout-mobile-docs/checkout-3ds-sdk-ios/index.html)
 
-### Integrating with any 3DS provider
+### Any 3DS provider
 Standalone 3DS allows our SDKs to be used with any Authentication provider, be it Checkout.com or otherwise. This is a higher touch integration that breaks up the 3DS flow more granularly. In order to integrate with the Standalone 3DS service:
 
 1. Initialize the SDK with your preferred user interface options using our `uiCustomization` object.
 2. Create `transaction` object 
 3. Get `authenticationRequestParameters` for the AReq
-4. If a challenge is mandated from the Authentication response from your 3DS server then call the `doChallenge` method to render the challenge however, if the challenge is not mandated by the ACS then it would have triggered a frictionless 3DS flow. 
+4. if a challenge is mandated from the Authentication response from your 3DS server then call the `doChallenge` method to render the challenge however, if the challenge is not mandated by the ACS then it would have triggered a frictionless 3DS flow. 
 
-#### Diagram
+#### End-to-end 3DS flow
 Here is a useful diagram that highlights the end-to-end 3DS flow using our `standalone3DSService`.
 ![E2E Standalone SDK Flow](https://user-images.githubusercontent.com/102961713/226956636-d39b7bff-9fb8-4701-a3bc-86d932b306f0.jpg)
 
 #### Code snippet 
 
-1. Creates an instance of `ThreeDS2Service` through which the 3DS Requestor App can create a transaction object to get the `authenticationRequestParameters` that are required to perform a challenge:
+1- Creates an instance of `ThreeDS2Service` through which the 3DS Requestor App can create a transaction object to get the `authenticationRequestParameters` that are required to perform a challenge:
 
 ```swift
    private var transaction: Transaction?
    private var standalone3DSService: ThreeDS2Service?
 ````
 
-2. Initialize the SDK with your preferred user interface options:
+2-Initialize the SDK with your preferred user interface options:
 
-- `scheme` can be set only as `visa` and `mastercard` in lowercase string.
+- `scheme` can be set only as `visa` and `mastercard` in lower case string.
```

Comment:
on the other PR, @patrick-hoban-cko also suggested to use lowercase btw. I think that PR comment was resolved without being resolved